lego/apps/users/serializers/abakus_groups.py

```diff
@@ -43,3 +43,9 @@ class Meta:
             'logo',
             'type',
         )
+
+
+class AbakusGroupNameSerializer(serializers.ModelSerializer):
```

Comment:
don't need this serialize anymore right?

Reply:
I'll remove it when I refactor and add tests later 👍